scenes/scene.yaml

```diff
@@ -80,10 +88,11 @@ styles:
 sources:
     osm:
         type: MVT
-        url:  https://vector.mapzen.com/osm/all/{z}/{x}/{y}.mvt
+        #url:  https://vector.mapzen.com/osm/all/{z}/{x}/{y}.json
+        mbtiles: nyc.mbtiles
```

Comment:
Having the tile url in the metadata would be an option to allow updating of externally created files. At the moment I'm adding the check to only write to the db when it contains `description:MBTiles tile container created by Tangram ES.` :)